ferret/evaluators/faithfulness_measures.py

```diff
@@ -285,7 +278,7 @@ def compute_evaluation(
             discrete_expl_ths.append(discrete_expl_th)
 
         if discrete_expl_ths == []:
-            return EvaluationMetricOutput(self.cls, 1)
+            return EvaluationMetricOutput(self, 1)
```

Comment:
could you explain this?

Reply:
When running the code in the [ReadME.md](http://readme.md/) under "Explain and Benchmark", it leads to the following error when computing evaluations : "AttributeError: 'AOPC_Sufficiency_Evaluation' object has no attribute 'cls'", since self.cls never gets set anywhere.
This change follows how this case is handled in the class "AOPC_Comprehensiveness_Evaluation(BaseEvaluator)" in line 142-143 where self, instead of self.cls, gets returned.
After applying the change, the code runs without errors.

Reply:
Cool